### ISSUE TITLE
twisted/application/reactors.py uses implements

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7161]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Alex reported|
|-|-|
|Trac ID|trac#7161|
|Type|enhancement|
|Created|2014-04-15 21:13:46Z|
|Branch|https://github.com/twisted/twisted/tree/implementer-7161|

It should use @implementer instead

<details><summary>Searchable metadata</summary>

```
trac-id__7161 7161
type__enhancement enhancement
reporter__Alex Alex
priority__normal normal
milestone__Python_3_x Python-3.x
branch__branches_implementer_7161 branches/implementer-7161
branch_author__pahan pahan
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1397596426000000 1397596426000000
changetime__1397601509000000 1397601509000000
version__None None
owner__PenguinOfDoom PenguinOfDoom

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|Alex commented|
|-|-|

This is fixed by https://github.com/twisted/twisted/pull/17/files

**Comment by twisted-trac** [comment]
|<img alt="pahan's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|pahan commented|
|-|-|

(In [42299]) Branching to implementer-7161.

**Comment by twisted-trac** [comment]
|<img alt="PenguinOfDoom's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|PenguinOfDoom set owner to PenguinOfDoom<br>PenguinOfDoom set status to `assigned`|
|-|-|

[Build Results](http://buildbot.twistedmatrix.com/boxes-supported?branch=%2Fbranches%2Fimplementer-7161)

**Comment by twisted-trac** [comment]
|<img alt="pahan's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|pahan set status to `closed`|
|-|-|

(In [42307]) Merge implementer-7161

Author: Alex
Reviewer: PenguinOfDoom
Fixes: [#7161](https://github.com/twisted/twisted/issues/7161)
